### PR TITLE
Add limit clause

### PR DESCRIPTION
**Pull request by dosht**
- [x] Add limit
- [ ] Tests

### CODE REVIEW

File: cql-parser/src/main/scala/troy/cql/parser/CqlParser.scala [hunk 1]
```diff
@@ -142,10 +142,12 @@ object CqlParser extends JavaTokenParsers with Helpers {
       "WHERE".i ~> rep1sep(relation, "AND".i) ^^ WhereClause.apply
     }
 
+    def limit = "LIMIT" ~> """([1-9]\d+)""".r ^^ { _.toInt }
```
Comment (tabdulradi) [inline cql-parser/src/main/scala/troy/cql/parser/CqlParser.scala]: `"LIMIT"` should be `"LIMIT".i`, which means "case **i**nsensitive"

Comment (tabdulradi) [inline cql-parser/src/main/scala/troy/cql/parser/CqlParser.scala]: Regarding the regex, I guess `"""\d+""".r` is enough. Also you might define it near the end of the files to be reused as `postiveNumber`.

**Comment by OmniaGM** [comment]
Hey dosht, 
I cherry pick your changes in my pull request where I refactored select statement based on based on CQL 3.4.3 
https://github.com/tabdulradi/troy/pull/44

**Comment by tabdulradi** [comment]
Merged as part of https://github.com/tabdulradi/troy/pull/44

**Pull request closed without merging**